lib/stdlib/doc/src/timer.xml

```diff
@@ -405,22 +486,76 @@ timer:cancel(R),
       functions
+      <seemfa marker="#apply_interval/2"><c>apply_interval/2</c></seemfa>,
+      <seemfa marker="#apply_interval/3"><c>apply_interval/3</c></seemfa>,
       <seemfa marker="#apply_interval/4"><c>apply_interval/4</c></seemfa>,
-      <seemfa marker="#send_interval/3"><c>send_interval/3</c></seemfa>, and
-      <seemfa marker="#send_interval/2"><c>send_interval/2</c></seemfa>
+      <seemfa marker="#apply_repeatedly/2"><c>apply_repeatedly/2</c></seemfa>,
+      <seemfa marker="#apply_repeatedly/3"><c>apply_repeatedly/3</c></seemfa>,
+      <seemfa marker="#apply_repeatedly/4"><c>apply_repeatedly/4</c></seemfa>,
+      <seemfa marker="#send_interval/2"><c>send_interval/2</c></seemfa>, and
+      <seemfa marker="#send_interval/3"><c>send_interval/3</c></seemfa>
       is linked to the process to which the timer performs its task.</p>
 
     <p>A one-shot timer, that is, a timer created by evaluating any of the
       functions
+      <seemfa marker="#apply_after/2"><c>apply_after/2</c></seemfa>,
+      <seemfa marker="#apply_after/3"><c>apply_after/3</c></seemfa>,
       <seemfa marker="#apply_after/4"><c>apply_after/4</c></seemfa>,
-      <seemfa marker="#send_after/3"><c>send_after/3</c></seemfa>,
       <seemfa marker="#send_after/2"><c>send_after/2</c></seemfa>,
-      <seemfa marker="#exit_after/3"><c>exit_after/3</c></seemfa>,
+      <seemfa marker="#send_after/3"><c>send_after/3</c></seemfa>,
       <seemfa marker="#exit_after/2"><c>exit_after/2</c></seemfa>,
-      <seemfa marker="#kill_after/2"><c>kill_after/2</c></seemfa>, and
-      <seemfa marker="#kill_after/1"><c>kill_after/1</c></seemfa>
+      <seemfa marker="#exit_after/3"><c>exit_after/3</c></seemfa>,
+      <seemfa marker="#kill_after/1"><c>kill_after/1</c></seemfa>, and
+      <seemfa marker="#kill_after/2"><c>kill_after/2</c></seemfa>
       is not linked to any process. Hence, such a timer is removed only
       when it reaches its time-out, or if it is explicitly removed by a call to
       <seemfa marker="#cancel/1"><c>cancel/1</c></seemfa>.</p>
+
+    <p>The functions given to 
+      <seemfa marker="#apply_after/2"><c>apply_after/2</c></seemfa>,
+      <seemfa marker="#apply_after/3"><c>apply_after/3</c></seemfa>,
+      <seemfa marker="#apply_interval/2"><c>apply_interval/2</c></seemfa>,
+      <seemfa marker="#apply_interval/3"><c>apply_interval/3</c></seemfa>,
+      <seemfa marker="#apply_repeatedly/2"><c>apply_repeatedly/2</c></seemfa>, and
+      <seemfa marker="#apply_repeatedly/3"><c>apply_repeatedly/3</c></seemfa>,
+      or denoted by <c>Module</c>, <c>Function</c> and <c>Arguments</c> given to
+      <seemfa marker="#apply_after/4"><c>apply_after/4</c></seemfa>,
+      <seemfa marker="#apply_interval/4"><c>apply_interval/4</c></seemfa>, and
+      <seemfa marker="#apply_repeatedly/4"><c>apply_repeatedly/4</c></seemfa>
+      are executed in an own process, and therefore calls to <c>self()</c> in
```

Comment:
"an own" feels strange to me:

```suggestion
      are executed in its own process, and therefore calls to <c>self()</c> in
```
or perhaps it would be better for clarity to write: "in a freshly-spawned process"